issue_07/script.js

```diff
@@ -1,11 +1,16 @@
-function loading() {
-    const div = document.getElementById('js-loading');
-    const loadingImg = document.createElement('img');
-    loadingImg.src = 'loading-circle.gif';
-    loadingImg.alt = 'ローディング画像';
+const div = document.getElementById('js-loading');
+const loadingImg = document.createElement('img');
+loadingImg.src = 'loading-circle.gif';
+loadingImg.alt = 'ローディング画像';
+
+function startLoading() {
     div.appendChild(loadingImg);
 }
 
+function stopLoading() {
+    div.removeChild(loadingImg);
+}
+
```

Comment:
I thought these function names should be easy to understand!

Reply:
Thanks for your comment!
I'm glad you said that because I wasn't too sure about the variable and function names.:sparkles: